init.rb

```diff
@@ -19,6 +12,7 @@
   requires_redmine :version_or_higher => '2.0.0'
```

Comment:
Does requerements of min version changes with usage of ActiveSupport::Reloader. It should be rails 5-specific?

Reply:
You are right, I made another commit, now it should work again with Redmine 3